charts/yurt-manager/templates/yurt-manager-auto-generated.yaml

```diff
@@ -632,6 +632,27 @@ webhooks:
     resources:
     - platformadmins
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      name: yurt-manager-webhook-service
+      namespace: {{ .Release.Namespace }}
+      path: /mutate-core-openyurt-io-v1-pod
+  failurePolicy: Ignore
+  name: mutate.core.v1.pod.openyurt.io
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
```

Comment:
Only intercept `CREATE` pod is enough, because of pods are managed by workloads like Deployment, so pods will be recreated if annotation is changed. please remove `UPDATE` operation.

Reply:
reasonable, `UPDATE` is redundant, fixed.